packages/tools/migrations-irec/config/seed.sql

```diff
@@ -664,4 +664,75 @@ VALUES (
         'd49f7fed-2f79-4d10-8985-41a0c3e9ba03',
         'METER003',
         'Biomass Facility description'
-    );
\ No newline at end of file
+    );
+
+INSERT INTO public."platform_organization" (
+    "createdAt",
+    "updatedAt",
+    id,
+    irecBeneficiaryId,
+    organizationId
+)
+VALUES (
+           '2020-03-30 09:55:25.962333+02',
+           '2020-03-30 09:55:25.962333+02',
+           1,
```

Comment:
Inserting that manually doesn't increment serial field, if this should be autoincrementing, which will cause conflicts if you want to insert anything later on. is this autoincrement field?

Reply:
fixed by adding `nextval`